pkg/cmd/roachtest/main.go

```diff
@@ -259,6 +263,9 @@ runner itself.
 		&teamCity, "teamcity", false, "include teamcity-specific markers in output")
 	runCmd.Flags().BoolVar(
 		&disableIssue, "disable-issue", false, "disable posting GitHub issue for failures")
+	runCmd.Flags().IntVar(
+		&promPort, "prom-port", 2112,
```

Comment:
The same default port is used by workload (in pkg/workload/cli/run.go). This means we can't run roachtest's test-runner and workload on the same host; while not a typical scenario, maybe we could use a distinct port, say `2113`?